linear_elasticity/linear_elasticity.cc

```diff
@@ -607,6 +553,73 @@ namespace Linear_Elasticity
   }
 
 
+  // Process RHS assembly, which is the coupling data (stress) in this case
+  template <int dim>
+  void
+  ElastoDynamics<dim>::assemble_consistent_loading(Vector<double> &dst)
```

Comment:
Follow-up of the previous comment.

```suggestion
  ElastoDynamics<dim>::assemble_consistent_loading()
```
We can then use `system_rhs` instead of `dst` in the function body.

If you wanted to test this method, then you could also setup the vector internally and return it by value.
As you change other members too, this may be a bit misleading though.
Ideal for testing are stand-alone functions that are independent of any global state aka pure functions.

```
  template <int dim>
  Vector<double>
  ElastoDynamics<dim>::assemble_consistent_loading();
```

Reply:
No, I don't want to test it. My first idea was to use some function pointer during the configuration since the `if` (see above) is now checked in each iteration rather than only once. The non-`const`ness of this function, however, disabled the utilization of lambdas and after a while I thought it's not worth spending more time here. So, it's just a deprecated layout.
Allocating a global vector in the function and returning it at the end  i.e. making a copy, sounds wrong though.